Sources/StreamChatUI/CommonViews/Suggestions/ChatSuggestionsVC.swift

```diff
@@ -233,7 +239,7 @@ open class ChatMessageComposerSuggestionsMentionDataSource: NSObject,
     }
 
     public func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
-        searchController.users.count
+        usersCache.count
     }
```

Comment:
Like I said above, the data should still come from the search controller. The search controller will decide if the data comes from remote or local